extensions/amp-story/1.0/amp-story-bling-link.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Bling link component that turns into a tooltip when clicked.
+ */
+
+import {Layout} from '../../../src/layout';
+
+/** @const */
+const HOVER_CLASS = 'i-amphtml-story-bling-link-hover';
+
+/** @const */
+const PULSE_CLASS = 'pulse';
+
+/** @const */
+const MATERIAL_ICON_CLASS = 'material-icons';
+
+/** @const */
+const ICON_CLASS = 'i-amphtml-story-bling-link-icon';
+
+export class AmpStoryBlingLink extends AMP.BaseElement {
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+  }
+
+  /** @override */
+  isLayoutSupported(layout) {
+    return layout == Layout.CONTAINER;
```

Comment:
My guess is we'd want these to be `Layout.FIXED`, since they're icons (and I assume we always show them at a fixed size)